src/topology/algebra/ordered/basic.lean

```diff
@@ -2256,6 +2259,24 @@ by rw [← Ici_inter_Iio, interior_inter, interior_Ici, interior_Iio, Ioi_inter_
 @[simp] lemma interior_Ioc [no_max_order α] {a b : α} : interior (Ioc a b) = Ioo a b :=
 by rw [← Ioi_inter_Iic, interior_inter, interior_Ioi, interior_Iic, Ioi_inter_Iio]
 
+lemma Icc_subset_closure_interior {a b : α} (h : a ≠ b) : Icc a b ⊆ closure (interior (Icc a b)) :=
```

Comment:
Can you turn this into an equality? (the other direction is `closure_minimal interior_subset is_closed_Icc`)